sklearn/discriminant_analysis.py

```diff
@@ -143,9 +143,6 @@ class LinearDiscriminantAnalysis(BaseEstimator, LinearClassifierMixin,
     .. versionadded:: 0.17
        *LinearDiscriminantAnalysis*.
 
-    .. versionchanged:: 0.17
-       Deprecated :class:`lda.LDA` have been moved to :class:`LinearDiscriminantAnalysis`.
```

Comment:
Do we want to remove this?

If not, similar comment as before `lda.LDA` instead of `:class:lda.LDA`...